apps/server/src/modules/sharing/service/share-token.service.ts

```diff
@@ -61,7 +63,13 @@ export class ShareTokenService {
 			case ShareTokenParentType.Task:
 				parentName = (await this.taskService.findById(shareToken.payload.parentId)).name;
 				break;
+			case ShareTokenParentType.ColumnBoard:
+				// TODO - well the column board should have a name...
+				parentName = (await this.columnBoardService.findById(shareToken.payload.parentId)).title;
```

Comment:
If it's empty than it's empty. I don't see a requirement to have the title set. If we are dealing with undefined just put `?? ''` in the end of the line.